release-notes/swan-lake-beta1-release-note.md

```diff
@@ -41,10 +41,33 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 ### Language Updates
 
-#### New Features
-
 #### Improvements
 
+- Worker deadlock detection has been improved to include the `wait` action.
+- `self` of an isolated object no longer needs to be accessed within a `lock` statement unless it is used to access a field that is either not `final` or is not a subtype of `readonly` or `isolated object`.
+- `xml:strip()` does not mutate the XML value, thus using `xml:strip()` on immutable XML values is allowed.
```

Comment:
```suggestion
- The `xml:strip()` does not mutate the XML value, and thereby, using the `xml:strip()` on immutable XML values is allowed.
```

Reply:
Same comment regarding "the". The content within backticks in these contexts usually refer to names, so we can't just say "The `x` does not...". I think we should use therefore instead of thereby here.

```suggestion
- The `lang.xml:strip()` function does not mutate an XML value. Therefore, using `lang.xml:strip()` on immutable XML values is allowed.
```

Reply:
+1